packages/vuetify/src/labs/VNumberInput/VNumberInput.tsx

```diff
@@ -39,18 +39,23 @@ const makeVNumberInputProps = propsFactory({
   hideInput: Boolean,
   min: {
     type: Number,
-    default: -Infinity,
+    default: Number.MIN_SAFE_INTEGER,
   },
   max: {
     type: Number,
-    default: Infinity,
+    default: Number.MAX_SAFE_INTEGER,
   },
   step: {
     type: Number,
     default: 1,
   },
 
-  ...omit(makeVTextFieldProps(), ['appendInnerIcon', 'prependInnerIcon']),
+  ...omit(makeVTextFieldProps({
+    modelValue: {
+      type: [Number, null],
```

Comment:
`type: null` means any. Vue always allows props to be null but doesn't include it in the types for some reason.

Reply:
Oh yeah propsFactory also only lets you set the default, this is basically
```
modelValue: {
  type: null,
  default: {
    type: [Number, null],
    default: null,
  }
}
```